content/docs/tutorials/best-practices-with-ocm.md

```diff
@@ -138,7 +137,7 @@ $ VERSION=0.1.0
 $ docker buildx build --load -t ${TAG_PREFIX}/simpleserver:0.1.0-linux-amd64 --platform linux/amd64 .
 [+] Building 54.4s (14/14) FINISHED
  => [internal] load build definition from Dockerfile                                                                          0.0s
- => => transferring dockerfile: 660B                                                                                          0.0s
+ => => transferring Dockerfile: 660B                                                                                          0.0s
```

Comment:
but that's a terminal output... did you just s/dockerfile/Dockerfile/? :D

Reply:
good catch. I reverted the capitalisation for all parts in the console output.